Library/Formula/input-source-switcher.rb

```diff
@@ -0,0 +1,20 @@
+class InputSourceSwitcher < Formula
+  homepage "https://github.com/vovkasm/input-source-switcher"
+  url "https://github.com/vovkasm/input-source-switcher/archive/cfca8149c6805b333910858a22aeeb124cbca903.zip"
```

Comment:
There are no tagged releases, but the code hasn't changed in six months so I'd consider that stable. Is the only way to get this accepted to find the original author and get them to tag it?